source/styles/recipe-page.css

```diff
@@ -20,6 +20,13 @@
   filter: brightness(75%);
 }
 
+#increase-scale,
+#decrease-scale {
+  border: none;
+  background: none;
+  cursor: pointer;
+}
+
```

Comment:
Could we also make the arrow buttons get brighter when hovered over, and get dimmer when clicked on using `:hover` and `:active` pseudo-classes? Also, if you are going to do this, you need to make the arrow buttons not completely black but a dark gray so that the `brightness` styling actually works.

Reply:
Sure, I'll add that in!